src/index.js

```diff
@@ -52,6 +52,7 @@ module.exports = options => ({
         sourceMapsCorrections[absolutePath],
         sourceMap
       )
+      delete errorWasThrown[absolutePath]
```

Comment:
@chinesedfan Yes, i tested it and it worked perfectly. The line won't be hit if there will be parse error since it will go to catch block below. Cleaning here is easier than cleaning in ```result```. Other module levels variables don't need cleaning because they're being initialized with fresh object/arrays literals before parsing, i.e:
```js
sourceMapsCorrections[absolutePath] = {}
interpolationLinesMap[absolutePath] = interpolationLines.concat(
        interpolationLinesMap[absolutePath] || []
)
```